source/adding_containers.rst

```diff
@@ -123,6 +123,15 @@ seen in the above code block, where we also set the
 attribute via this mechanism.
 
 
+.. Caution::
+
+   The current setup **cannot** replace versions in READMEs. If you rely on
```

Comment:
we only have this limitation for tumbleweed/opensuse containers? the others should be working fine?

it feels like yet another cumbersome thing to maintain going forward.

Reply:
> we only have this limitation for tumbleweed/opensuse containers? the others should be working fine?

This is an issue for **every** container. The replace_using_package_version service replaces the variables in the README **in the buildroot**. But the README that we "ship" is not the one from inside the container, but the one in the source package. That one is not modified by replace_using_package_version (and also cannot be).

> it feels like yet another cumbersome thing to maintain going forward. did anyone report this as an issue so far?

I talked about this last week with darix and there's really no easy solution that we came up with.